internal/pkg/manifest/http.go

```diff
@@ -47,37 +53,47 @@ func (r *RoutingRuleConfigOrBool) UnmarshalYAML(value *yaml.Node) error {
 	return nil
 }
 
-// RoutingRuleConfiguration holds the path to route requests to the service.
+// RoutingRuleConfiguration holds options for application load balancer.
 type RoutingRuleConfiguration struct {
+	Main                     RoutingRule   `yaml:",inline"`
+	TargetContainerCamelCase *string       `yaml:"targetContainer"` // Deprecated. "targetContainerCamelCase" for backwards compatibility
```

Comment:
Wut is this comment malformed? what is "targetContainerCamelCase" should this be:
```suggestion
	TargetContainerCamelCase *string       `yaml:"targetContainer"` // Deprecated. Maintained for backwards compatibility, use [RoutingRule.TargetContainer] instead.
```